syntaxFilters.js

```diff
@@ -64,65 +64,45 @@ function headsOnWrongSide(sTree, side, strict){
 
 }
 
-// Input Gen option to remove trees that are mirror images of trees earlier in the list
-// returns true if tree is a mirror of an eariler tree
-// returns false if tree is not a mirror image of an earlier tree
+var isMirror = true;
```

Comment:
I'm worried about strange scope issues arising from declaring var isMirror out here. I think it would be better to declare it inside mirrorImages(), pass it to checkMirror(), and return it to mirrorImages().

Reply:
I agree, this was an admittedly hacky fix.